code/__DEFINES/layers.dm

```diff
@@ -154,8 +154,8 @@
 #define HUD_LAYER 22
 #define HUD_RENDER_TARGET "HUD_PLANE"
 #define ABOVE_HUD_PLANE 23
-#define ABOVE_HUD_LAYER 23
```

Comment:
Why was this even a thing to begin with? everything already defaults to being above the hud layer